markdown/report.Rmd

```diff
@@ -0,0 +1,416 @@
+
+More recently in December 2016 the ["Health and Social Care Delivery Plan"](http://www.gov.scot/Resource/0051/00511950.pdf) was launched, which reinforces key actions and services to respond to palliative and end of life care needs among other health and social care priorities.
+
+### Provisional Figures 
+
+Note that hospital data and National Records of Scotland deaths data are provisional for 2018/19. This publication will be updated in October 2019 with final figures.  Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year.  This is not expected to have a large impact on these published figures when they are revised in subsequent updates. See [Appendix 2](#Appendix2) for more information.
+
+
+<Div custom-style = "Heading 2_pgbrk">Main Points</div>
+
+*	In 2018/19, there were 53,168 deaths in Scotland excluding those where an external cause such as unintentional injury was recorded. For these individuals, 89.2% of their last six months of life was spent either at home or in a community setting, with the remaining 10.8% spent in hospital. This is equivalent to an average of 20 days in hospital in the six months prior to death.
+
+*	Over the past nine years, the percentage of time spent at home or in a community setting has gradually increased from 85.3% in 2010/11 to 89.2% in 2018/19.
+
+*	In 2018/19, females aged 85 years and over spent a higher percentage of their last six months of life at home or in the community (90.0%) compared to males (87.8%); this equates to females spending, on average, an extra four days in the community in their last six months of life.
+
+<Div custom-style = "Heading 2_pgbrk">Results and Commentary</div>
+
+For the 53,168 residents of Scotland who died in financial year 2018/19, 89.2% of their last six months of life was spent either at home or in the community (Table 1) and 10.8% of their last six months of life was spent in hospital. This is equivalent to each individual spending an average of 20 days in hospital in the six months prior to their death (Table 1 in the [data tables](http://www.isdscotland.org/Health-Topics/Health-and-Social-Community-Care/Publications/2019-05-28/QOM_tables_201905.xls)).
+
+Over the nine years, 2010/11 to 2018/19 the percentage of time spent at home or in a community setting has increased from 85.3% to 89.2%, this equates to, on average an extra seven days within the last six months of life being spent at home or in a community setting in 2018/19 compared to 2010/11 (see Figure 1 below or [Appendix A1](#AppendixA1)).
+
+##### Figure 1: Percentage of last six months of life spent in hospital and at home or in a community setting; 2010/11 - 2018/19
+
+##### Table here
+
+1.	Figures for 2018/19 are incomplete and will be revised in October 2019 publication.  See Appendix 2 for more information.
```

Comment:
Use Normal_Notes style for notes:

<Div custom-style = "Normal_Notes">
1. Figures for 2018/19 are incomplete and will be revised in October 2019 publication.  See Appendix 2 for more information.
</div>